workshop/docker/data/records/example_catalog.geojson

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
Where we are we using this file?  Should we document it in `workshop/data/README.md`?

Reply:
Sorry, I've added this by mistake... This is for the OGC API Records. I was thinking to create a single geojson with all the records, to have everything in a single collection. But this is still a work in progress, removing this for now. Thanks